pom.xml

```diff
@@ -25,15 +26,13 @@
 
     <!-- jenkins -->
     <jenkins.version>1.642.3</jenkins.version>
-    <hpi-plugin.version>1.95</hpi-plugin.version>
```

Comment:
must not override

Reply:
This was purpose, I remember I was trying to bump it but there were some docs/lint checks IIRC that made the build failed (and indeed the build is now failing :-) )

Reply:
@jglick We could remove `<project.build.sourceEncoding>` since it is inherited as well.

Reply:
> indeed the build is now failing

Because you were trying to build using JDK 7, which is no longer supported. (And not necessary.)

Reply:
> We could remove `<project.build.sourceEncoding>` since it is inherited as well.

Right, will do.